app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -242,7 +242,7 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
             variant={ButtonVariant.danger}
             className="top-spacer"
             onClick={() => {
-              const regex = /\[.*\]/; // TODO: This is far too eager and could remove value content.
+              const regex = /\[.*\]/; // TODO: This is far too eager and could remove valuable content.
```

Comment:
Wonder if we should add a story to capture this, I like the way GitHub does their tags for PRs

Reply:
@jdtoombs Yes create a story for this.